render.yaml

```diff
@@ -0,0 +1,34 @@
+databases:
+  - name: maybe
+    user: maybe
+    plan: free
+
+services:
+  - type: web
+    plan: free
+    autoDeploy: false
+    runtime: ruby
+    name: maybe
+    repo: https://github.com/maybe-finance/maybe.git
+    branch: main
+    healthCheckPath: /up
+    buildCommand: "./bin/render-build.sh"
+    # preDeployCommand: "bundle exec rails db:migrate" # We run this in the startCommand instead because `free` plan doesn't allow this option in Blueprint
+    startCommand: "bundle exec rails db:migrate && bundle exec rails server"
```

Comment:
I'd suggest moving the db:migrate into the build script - you probably don't want the DB migrate task running for every start of the service

Reply:
That's true, I'll go ahead and make that change.

Ideally this is run in `preDeployCommand`, so I may add something like this to the `render-build.sh` script:

```sh
# Self-hosting users who upgrade their service should remove the environment variable RUN_DB_MIGRATIONS_IN_BUILD_STEP
# since the database will be migrated in the `preDeployCommand` (a premium Render feature)
if [ "$RUN_DB_MIGRATIONS_IN_BUILD_STEP" = "true" ]; then
  echo "Running migrations for free tier instance..."
  bundle exec rails db:migrate
fi

```